app/globals.css

```
@@ -555,4 +556,7 @@ input[type="range"]::-moz-range-progress {
   outline: none;
   min-height: 100%;
   cursor: default;
+  writing-mode: horizontal-tb;
+  word-wrap: break-word;
```

Stylelint (17.3.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)